lib/src/meta.dart

```diff
@@ -99,3 +99,66 @@ void _forEachGeomInGeometryObject(
     throw Exception('Unknown Geometry Type');
   }
 }
+
+/// Callback for propEach
+typedef PropEachCallback = dynamic Function(
+    Map<String, dynamic> currentProperties, num featureIndex);
+
+/// Iterate over properties in any [geoJSON] object, calling [callback] on each
+/// iteration. Similar to Array.forEach()
+///
+/// For example:
+///
+/// ```dart
+/// FeatureCollection featureCollection = FeatureCollection(
+///   features: [
+///     point1,
+///     point2,
+///     point3,
+///   ],
+/// );
+/// propEach(featureCollection, (currentProperties, featureIndex) {
+///   someOperationOnEachProperty(currentProperties);
+/// });
+/// ```
+void propEach(GeoJSONObject geoJSON, PropEachCallback callback) {
+  var i;
+  if (geoJSON is FeatureCollection) {
+    for (var i = 0; i < geoJSON.features.length; i++) {
+      if (callback(geoJSON.features[i].properties!, i) == false) break;
```

Comment:
this more relates to `turf_dart`s handling of null values, but if a geoJSON object doesn't have a props, but get's passed in here, this will just throw a runtime error because there is no meaningful null checks.

In this specific instance, would it better to silently throw away the props object and either break or continue?

The other option (which I am leaning towards as the correct option) is to make the callback function parameter nullable and let the user deal with it, since `A Feature object has a member with the name "properties".  The value of the properties member is an object (any JSON object or a JSON null value).` ([3.2](https://datatracker.ietf.org/doc/html/rfc7946#section-3.2)) and we decided to make it nullable.

Reply:
I just filed another issue that deals with a similar problem in `geomEach` implementation: https://github.com/dartclub/turf_dart/issues/36

Reply:
> which I am leaning towards as the correct option

I went with the nullable callback